### PR TITLE
#66 change readme

### LINKED ISSUE #66: Change README.md with basic info and link to github.io page.

### PR DESCRIPTION
**Pull request by crsanti**
This fixed #66 .

### CODE REVIEW

**Comment by brauliodiez** [comment]
Good stuff, almost there !!! Just only pending the ES6 sample and we are good to go.

Could you create an entry in the github io page giving tips on how to implement a multilanguage approac (just based on the TYPE field), this can server as an entry point for @luisgonzalo to implement the sample

**Comment by crsanti** [comment]
Do you mean by using a service (that gets the message from a resource) and pass the `type` to get the error message??

**Comment by brauliodiez** [comment]
Just guidance, in the github io just add one section / subsection where we talk about multinlanguage support, providing some guidance about how to impement it in an app (but just create the issue), let's focus on closing the 1.0 version

**Comment by crsanti** [comment]
ok!

File: README.md [hunk 1]
```diff
@@ -1,177 +1,300 @@
...
+  })
+  .catch((error) => {
+    // handle unexpected errors
+  });
+```
 
-# FormBaseValidation + Sample Form #
+### Trigger certain validations in each constraint:
+
+Add `eventsFilter` with on or more events:
```
Comment (nasdan) [inline README.md:90]: Typo `with one or more`?

Comment (crsanti) [reply README.md:90]: And there is my boy :clap:

**Pull request merged**